src/systems/odometry_publisher/OdometryPublisher.cc

```diff
@@ -92,10 +92,6 @@ class gz::sim::systems::OdometryPublisherPrivate
   public: std::tuple<math::RollingMean, math::RollingMean, math::RollingMean>
     linearMean;
 
-  /// \brief Rolling mean accumulators for the angular velocity
```

Comment:
I think this change in behavior can be risky since users depending on the published angular velocity to be reasonably smooth might be broken by non-smooth values coming out of the finite difference on line 370. Could you kindly add the filter back? I'd suggest to open a follow up PR that accepts a moving average window size parameter in the System's xml input that the user can use to configure the size of the filter for both linear and angular velocities.

Reply:
Fixed in [10082f9](https://github.com/gazebosim/gz-sim/pull/2348/commits/10082f9ffd9989abcd80546b5e8d682d444eda3f)